doc/Technical Documentation/IEEE-695 Object File format.md

```diff
@@ -212,38 +215,39 @@ gives attributes for the section, and its name.  The linker may use the
 attributes and name, with possibly some other input, to combine and relocate 
 sections when generating the output.  It has the format:
 
->    **ST#,attributes,name.**
+```
+ST#,attributes,name.
+```
+
 
+Attributes are separated by a comma and are as follows: 
 
-Attributes are separated by ',' and are as follows: 
+* `'A'`: an absolute section not subject to relocation.
+  * When this attribute is present an ASL record must be present in the input file telling exactly where the section is located. 
 
-**'A'**: an absolute section not subject to relocation.
-When this attribute is present an ASL record must be present in the input 
-file telling exactly where the section is located. 
+* `'B'`: a section made up of bit values
+  * This attribute is reserved for processors like the C167 or 8051. 
```

Comment:
How would you do such a thing? I can't see raw markdown on github, and yes that would look nice but would be inconsistent with the other listed items and I valued consistency over anything else in this endevor.

Reply:
> How would you do such a thing?

Exactly like I did ?

```
### `C` - a common section. 
Two common sections with the same name are overlayed on each other at link time.  
Non-common sections normally follow one another. 
```

again: the line break is commonly rendered for MD if you have the previous line ended with two spaces and a line break.

> would be inconsistent with the other listed items and I valued consistency over anything else in this endevor.

That is the big issue here - I've suggested to have it consistent done this way in the complete document (_he said and ran away_)

Reply:
Hmm, looks nice for the most part and having headings you could just link to or go directly to from a ToC would be great, I'll try that in the next few days (I'm already tackling the other problems you've listed)

Reply:
To drop this here, too: my other suggestion to replace
```
* `'A'`: an absolute section not subject to relocation.
  * When this attribute is present an ASL record must be present in the input file telling exactly where the section is located.
* `'B'`: a section made up of bit values
  * This attribute is reserved for processors like the C167 or 8051. 
* `'C'`: A common section. 
  * Two common sections with the same name are overlayed on each other at link time. 
  * Non-common sections normally follow one another. 
```
by
```
* `'A'`: an absolute section not subject to relocation.
   When this attribute is present an ASL record must be present in the input file telling exactly where the section is located.
* `'B'`: a section made up of bit values
   This attribute is reserved for processors like the C167 or 8051.
* `'C'`: A common section. 
   Two common sections with the same name are overlayed on each other at link time.  
   Non-common sections normally follow one another. 
```
 would look like this
--------------------------
* `'A'`: an absolute section not subject to relocation.
   When this attribute is present an ASL record must be present in the input file telling exactly where the section is located.
* `'B'`: a section made up of bit values
   This attribute is reserved for processors like the C167 or 8051.
* `'C'`: A common section. 
   Two common sections with the same name are overlayed on each other at link time.  
   Non-common sections normally follow one another. 
--------------------------
instead of
--------------------------
* `'A'`: an absolute section not subject to relocation.
  * When this attribute is present an ASL record must be present in the input file telling exactly where the section is located.
* `'B'`: a section made up of bit values
  * This attribute is reserved for processors like the C167 or 8051. 
* `'C'`: A common section. 
  * Two common sections with the same name are overlayed on each other at link time. 
  * Non-common sections normally follow one another. 


side note: either each entries should start with a capital and end with a period (like "A common section.") or should just be an entry (like "an equal section") [I personally prefer the later]

Reply:
I made that change you requested for the new suggestion, check it out and see what I might need to change please